manim/camera/three_d_camera.py

```diff
@@ -85,7 +85,7 @@ def capture_mobjects(self, mobjects, **kwargs):
         super().capture_mobjects(mobjects, **kwargs)
 
     def get_value_trackers(self):
-        """Returns list of ValueTrackers of phi, theta, focal_distance and gamma
+        """Returns list of ValueTrackers of phi, theta, focal_distance, gamma and zoom
```

Comment:
Minor improvement regarding language and documentation formatting.

```suggestion
        """A list of :class:`ValueTrackers <.ValueTracker>` of phi, theta, focal_distance,
        gamma and zoom.
```

Reply:
That is nicer, btw is there a maximum line width for docstrings?
In the documentation it mentions the numpy one which does have one, but not all the docstrings in this file follow that and even the example docstring doesnt.